addons/forward/models.py

```diff
@@ -35,17 +32,6 @@ def after_register(self, node, registration, user, save=True):
 
         return clone, None
 
-    def save(self, request=None, *args, **kwargs):
```

Comment:
Why was this removed? If it goes from `not spammy url` to `spammy url`, we probably still want to be sniffing it.